lib/metrics.js

```diff
@@ -158,6 +158,24 @@ class AssetComputeMetrics {
         }
     }
 
+    /**
+     * Immediately sends `activation_start` metrics
+     *
+     * Call this when your openwhisk action has started. Does nothing on multiple calls
+     *
+     * @param {Object} metrics all custom metrics to be included in the New Relic event
+     * @param {Boolean} sendActivationStartMetrics if false, it will not send `activation_start` metrics
+     */
+    async activationStarted(metrics={}, sendActivationStartMetrics=true) {
```

Comment:
Also, mention this should never ever throw exceptions (because `this.sendMetrics` should not). This is important because the `Promise.all` used for the activation call would reject if any promise fails.